Source/DafnyCore/DafnyOptions.cs

```diff
@@ -360,6 +360,7 @@ public enum ContractTestingMode {
     public int DeprecationNoise = 1;
     public bool VerifyAllModules = false;
     public bool SeparateModuleOutput = false;
+    public bool EmitUncompilableCode { get; set; }
```

Comment:
New options shouldn't be added here, they should just use the new `Option` model

Reply:
I am not aware of any other mechanism to read options of the new CLI other than assigning them to this object. Can you please point me to the documentation of what you suggest? I tried at call site things like

    options.Options.Arguments[CommonOptionBag.EmitUncompilableCode]

but none of my versions work, and I don't know where to find an example of what you have in mind. Your lights are appreciated.